compute/kubernetes/api-cli/creating-managing-kubernetes-lifecycle-cliv2.mdx

```diff
@@ -0,0 +1,322 @@
+  - You have generated your [API key](/console/my-project/how-to/generate-api-key/)
+  - You have downloaded and installed the [Scaleway CLI](https://github.com/scaleway/scaleway-cli#scaleway-cli-v2)
+  - You have configured the [Scaleway CLI](/compute/bare-metal-cloud/api-cli/bare-metal-with-cli/#-Configuring-the-CLI)
+
+</Message>
+
+
+## Configuring the CLI
+
+1. You must configure the Scaleway CLI on your machine. Open a terminal and type the following command:
+
+  `scw init`
+  
+  If you have already configured the CLI, you will see a message like the following:
+
+  ```
+  Current config is located at /path/to/config.yaml
+  access_key: <YOUR ACCESS KEY>
+  secret_key: <YOUR SECRET KEY>
+  default_organization_id: <YOUR ORGANIZATION ID>
+  default_project_id: <YOUR PROJECT ID>
+  default_region: <YOUR DEFAULT REGION, eg fr-par>
+  default_zone: <YOUR DEFAULT ZONE, eg fr-par-1>
+
+  Do you want to override the current config? (Y/n):  
+  ```
+
+2. Unless you want to change your configuration, you can type `n` and go directly to the [Creating a K8s Kapsule cluster](#creating-a-k8s-kapsule-cluster) step of this tutorial.
+
+  If you have not yet configured the CLI, you will be guided through the configuration process via a series of questions:
+
+  ```
+  Enter a valid secret-key or an email
+  ```
+
+3. Enter either the email address for your account (you will then be asked for your password), or your [API Secret Key](/console/my-project/how-to/create-ssh-key/):
+
+  ```
+  To improve this tool we rely on diagnostic and usage data.
+  Sending such data is optional and can be disabled at any time by running "scw config set send-telemetry=false".
+  Do you want to send usage statistics and diagnostics? (Y/n):
+  ```
+
+4. Type `y` or `n` and hit enter.
+
+  ```
+  To fully enjoy Scaleway CLI we recommend you install autocomplete support in your shell.
+  Do you want to install autocomplete? (Y/n):
+  ```
+5. Type `y` or `n` and hit enter.
+
+  ```
+  To enable autocomplete, scw needs to update your shell configuration.
+  What type of shell are you using (default: zsh):
+  ```
+
+6. Hit enter if the default shell type is already correct for you. Otherwise, enter your shell type and hit enter.
+
+  ```
+  To enable autocomplete we need to append to /Users/yourusername/.zshrc the following lines:
+  # Scaleway CLI autocomplete initialization.
+  eval "$(scw autocomplete script shell=zsh)"
+  Do you want to proceed with these changes? (Y/n):
+  ```
+
+7. Type `y` or `n` and hit enter. This should complete the configuration process:
+
+  ```
+  Initialization completed with success.
+  ```
+
+## Creating a K8s Kapsule cluster
+
+1. Type the following command in your terminal to create a [K8S Kapsule cluster](/compute/kubernetes/concepts/#cluster/):
+
+  `scw k8s cluster create name=name-of-your-cluster version=1.22.1 pools.0.name=name-of-your-pool pools.0.node-type=GP1_XS pools.0.size=5`
```

Comment:
You can skip the version, that way the command will continue to work even when 1.22.1 will be deprecated.

Reply:
If you explain how to add a pool down below you can even simplify this command and just do: `scw k8s cluster create`